dashboard/test/controllers/home_controller_test.rb

```diff
@@ -194,4 +192,38 @@ def setup_user_with_gallery
       get :debug
     end
   end
+
+  test 'workshop organizers see only new dashboard links' do
```

Comment:
The "only" part makes me expect a `refute_select 'h1', text: 'Old Workshop Dashboard'`. Also in some of the below tests.